blockchain/blockchain-wallet-v4-frontend
# feat(1click): implemented way to set interest deposit form

## Description (optional)
This PR brings 1-click interest improvement that we fill up interest deposit form

## Testing Steps (optional)
- Tier 2 account
- go over simple buy and buy some crypto for $300
- once when you click on banner interest form should be pre-fill with data
